### PR TITLE
removed designer from the editor list and moved network preparations …

### PR DESCRIPTION
**Pull request by kaikreuzer**
…to the top as this is relevant for all editors

Signed-off-by: Kai Kreuzer <kai@openhab.org>

### CODE REVIEW

File: configuration/editors.md [hunk 1]
```diff
@@ -15,6 +15,17 @@ This documentation page can give you some guidance in choosing the right one for
 - TOC
 {:toc}
 
+### Network Preparations
+
+The editors need to be able to access the configuration files on the remote openHAB host.
```
Comment (bgilmer77) [inline configuration/editors.md]: Consider, "Any editors used to configure openHAB need to be able to..."

File: configuration/editors.md [hunk 2]
```diff
@@ -15,6 +15,17 @@ This documentation page can give you some guidance in choosing the right one for
...
+
+You have to have a [network share](https://en.wikipedia.org/wiki/Shared_resource) set up on the remote host and mounted on your local computer.
```
Comment (bgilmer77) [inline configuration/editors.md]: Consider, "This can be achieved by using a `network share`...

File: configuration/editors.md [hunk 3]
```diff
@@ -15,6 +15,17 @@ This documentation page can give you some guidance in choosing the right one for
...
+The required steps to do so are specific to the host operation system.
```
Comment (bgilmer77) [inline configuration/editors.md]: Consider, "The steps required to set up a `network share` on your local host computer are specific to..."

**Comment by kaikreuzer** [comment]
Thanks, @bgilmer77! I have addressed your comments.

**Comment by ThomDietrich** [comment]
@kaikreuzer a bold but imho good step forward. Thanks!

File: configuration/editors.md [hunk 4]
```diff
@@ -34,60 +45,10 @@ You can find it in the [Microsoft Visual Studio Marketplace](https://marketplace
...
-### Usage
-
-You can now start to create and modify your configuration files.
-Changes are automatically loaded by the openHAB runtime.
-More details about the syntax of the different configuration files can be found in the [Configuration]({{base}}/configuration/index.html) chapter.
-
-Hint: Press `<Ctrl + N>` to create a new file.
-
 {: #others}
 ## Other Editor Integrations
```
Comment (Nadahar) [inline configuration/editors.md]: `like the two tools above` below doesn't make sense after Designer was removed.

**Comment by Nadahar** [comment]
Is there any information somewhere about why Designer was removed? I just started with z-wave and opted to check out openHAB first since I'm very familiar with Java and Eclipse, which theoretically should make it relatively easy for me to contribute if I find something missing or wrong.

I was thus very disappointed when I discovered that VS Code was the only "supported IDE". I just can't stand VS in general and VS Code in particular, with it's "metro" UI and general lack of functionality. I took it for granted that I could use Eclipse for a project that is build on Java and is so tightly related to the Eclipse organization.

I assume that there's a reason for removing it from the documentation other than simply "taste". Has the designer been deprecated, abandoned or does it lack something fundamental? I can see that it hasn't been updated since December 2017, so I assume there's something here.

I've installed them both, and from the little I see at the level I am now the Designer is miles better to work with, both because it has a usable UI and because it has sensible "completion"/"intellisense". VS Code doesn't show me what valid options I have in a particular position, instead it seems to list all kind of stuff that is more or less relevant. Designer on the other hand does what I'd expect.

Can someone shed some light on this or link to where this information is available? Shouldn't the documentation also at least mention this, or is Designer "so useless" that it should not even be mentioned?

**Comment by Confectrician** [comment]
Designer has been deprecated long ago and isn't continued anymore.
For detailed reasons i would ask you to check the openHAB community forum.
I think there should be some threads about this.

That's also the reason for removing designer from the documentation.
The code extension is the currently recommended editor solution.
(Which is clearly some steps away from being perfect, to be honest.)

**Pull request merged**